packages/dev/parcel-resolver-docs/DocsResolver.js

```diff
@@ -33,7 +33,7 @@ module.exports = new Resolver({
       if (resolved) {
         // HACK: ensure source code is used to build types, not compiled code.
         // Parcel removes the source field from package.json when the code comes from node_modules.
-        if (resolved.filePath.endsWith('.d.ts') && !resolved.filePath.includes('@react-types')) {
+        if (/^@(react-spectrum|react-aria|react-stately|internationalized|react-types|spectrum-icons|adobe\/react-spectrum)/g.test(resolved.filePath) && resolved.filePath.endsWith('.d.ts') && !resolved.filePath.includes('@react-types')) {
```

Comment:
Do we need to include the aria and stately monopackages here as well? We don't use them in the docs I don't think but wanted to confirm.

Reply:
I think it's ok to keep them, then we'll know if those monopackages get updated in undesireable ways as well

Reply:
Oh I meant they are not in this list right? Only the `@react-aria` and `@react-stately` scope packages but not the monopackages (except `@adobe/react-spectrum`)
All assuming I am reading the regex correctly

Reply:
o, yep, you are right, i read your comment the opposite of how you expected me to, in that I assumed they were already included (b/c regex) and you were questioning if they should be
which reinforces the opinion, they should be included :)